recipe/meta.yaml

```diff
@@ -39,7 +39,7 @@ requirements:
     - libgsasl
```

Comment:
in build section the windows variant is missing for patch ... please add.

Reply:
please remove the ignore_run_exports for libntlm ... add an abs.yaml file disabling overdepending checks instead

Reply:
We skip building on `win` that's why there is only unix `patch`